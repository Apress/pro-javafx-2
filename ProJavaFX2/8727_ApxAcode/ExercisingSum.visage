/*
 * @test
 * @compilefirst Sum.java
 * @compilefirst SumClient.java
 * @compile VisageSum.visage
 * @run
 */

// ExercisingSum.visage
var visageSum = VisageSum {};
var sumClient = new SumClient(visageSum);
var sumOfInts = sumClient.addUpInts([1, 3, 5, 7, 9]);
var sumOfDoubles = sumClient.addUpDoubles([3.14159, 2.71828]);
println("sumOfInts={sumOfInts}.");
println("sumOfDoubles={sumOfDoubles}.");
