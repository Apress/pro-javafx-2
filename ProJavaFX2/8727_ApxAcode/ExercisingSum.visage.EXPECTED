sumOfInts=25.
sumOfDoubles=5.859870195388794.
